server/message_hooks.go

```diff
@@ -97,22 +170,26 @@ func (p *Plugin) MessageHasBeenPosted(_ *plugin.Context, post *model.Post) {
 
 		_, err = p.Send(link.MSTeamsTeam, link.MSTeamsChannel, user, post)
 		if err != nil {
-			p.API.LogWarn("Unable to handle message sent", "error", err.Error())
+			p.API.LogWarn("Unable to handle message sent -- Send", "error", err.Error())
 		}
 	}
 }
 
 func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
-	if !p.getConfiguration().SyncReactions {
+	if p.getConfiguration().UseSharedChannels {
 		return
 	}
+	p.reactionAddedHandler(reaction)
+}
 
-	updateRequired := true
-	if c.RequestId == "" {
```

Comment:
What was the purpose of the `c.RequestId` check before?

Reply:
It was used to differenciate requests that comes from a user interaction through the api or an internal call. It is not longer differenciable so I'm checking always the IgnorePluginHooksMap. Anyway, this is possible that it is not needed for shared channels, we can verify that later and check if we can remove it atfer removing the hooks.